Description: gumbo python module is trying to load libgumbo.so
 libgumbo1 package is providing libgumbo.so.1
 This patch is making gumbo python module to load libgumbo.so.1

--- a/python/gumbo/gumboc.py
+++ b/python/gumbo/gumboc.py
@@ -39,12 +39,8 @@
   _dll = ctypes.cdll.LoadLibrary(os.path.join(
       os.path.dirname(__file__), '..', '..', '.libs', _name_of_lib))
 except OSError:
-  # PyPI or setuptools install, look in the current directory.
-  _dll = ctypes.cdll.LoadLibrary(os.path.join(
-      os.path.dirname(__file__), _name_of_lib))
-except OSError:
   # System library, on unix or mac osx
-  _dll = ctypes.cdll.LoadLibrary(_name_of_lib)
+  _dll = ctypes.cdll.LoadLibrary(_name_of_lib + '.2')
 
 # Some aliases for common types.
 _bitvector = ctypes.c_uint
